Jenkinsfile

```
@@ -534,9 +534,18 @@ pipeline {
 
         stage('On a master build') {
           when { branch "master" }
-
+          // Parallel publish edge tag image
```

I think
```suggestion
          // Parallel publish edge tag image
```
I think that this comment is redundant but if you do write it please change it to
```
// Publish edge tag image to private and public registries in parallel
```